imports/plugins/core/ui/client/components/notFound/notFound.js

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import PropTypes from "prop-types";
+import classnames from "classnames";
+import { Components, registerComponent } from "@reactioncommerce/reaction-components";
+import { Icon } from "../icon";
+
+/**
+ * React component for displaying a not-found view
+ * @param {props} props React props
+ * @return {Node} React node containing not-found view
+ */
+const NotFound = (props) => {
+  // ClassName for base wrapper,
+  // If one is provied in props, the default is not used
+  const baseClassName = classnames({
+    "container-fluid-sm": typeof props.className === "undefined"
+  }, props.className);
+
+  // ClassName for content wrapper
+  // If one is provied in props, the default is not used
+  const contentClassName = classnames({
+    "empty-view-message": typeof props.contentClassName === "undefined"
+  }, props.contentClassName);
+
+  return (
+    <div className={baseClassName}>
+      <div className={contentClassName}>
+        { props.icon &&
+          <Icon icon={props.icon} />
+        }
+
+        { props.title &&
+          <Components.Translation defaultValue={props.title} i18nKey={props.i18nKeyTitle} />
+        }
+
+        { props.message &&
+          <Components.Translation defaultValue={props.message} i18nKey={props.i18nKeyMessage} />
+        }
+
+        {props.children}
+      </div>
+    </div>
+  );
+};
+
+NotFound.propTypes = {
+  /**
+   * Children
+   * @type {ReactNode}
```

Comment:
Should we make this more consistent? I feel like we call this different things different place (e.g. Node/ReactNode/JSX. Is there a difference?)